lib/postcode_validation/use_case/validate_address.rb

```diff
@@ -35,6 +37,11 @@ def potential_address_matches
         @address_match_gateway.query(search_term: postcode,
                                      country: country)
       end
+
+      def invalid_postcode_format?
+        validator = FormatValidator.build(country)
```

Comment:
maybe instead of `.build` it could be `.for`. I rage against the java conventions